kDriveCore/FileProvider/FileProviderItem.swift

```diff
@@ -124,6 +125,7 @@ public final class FileProviderItem: NSObject, NSFileProviderItem {
     init(file: File, parent: NSFileProviderItemIdentifier? = nil, drive: Drive?, domain: NSFileProviderDomain?) {
         Log.fileProvider("FileProviderItem init file:\(file.id)")
         @InjectService var fileProviderService: FileProviderServiceable
+        @LazyInjectService var downloadQueue: DownloadQueueable
```

Comment:
Here we are in a function as well, no need for lazy
```suggestion
        @InjectService var downloadQueue: DownloadQueueable
```